sdk/communication/communication-alpha-ids/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 - APIs for querying and managing the usage of Pre-registered Alpha IDs
+- Added environment variable `AZURE_USERAGENT_OVERRIDE` that overrides the HTTP header `x-ms-useragent` during tests
```

Comment:
Since this is only relevant to sdk developers, I wouldn't add it to the changelog. 

However, do add under bug fixes that paging was fixed.